cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -45,30 +44,32 @@ func RelayProphecyClaimToEthereum(provider string, contractAddress common.Addres
 	}
 
 	// Send transaction
-	sugaredLogger.Infow("Sending new ProphecyClaim to CosmosBridge.",
-		"CosmosSender", claim.CosmosSender,
-		"CosmosSenderSequence", claim.CosmosSenderSequence)
-	claimType, err := strconv.Atoi(claim.ClaimType)
-	if err != nil {
-		sugaredLogger.Errorw("failed convert ClaimType", errorMessageKey, err.Error())
-		return err
-	}
-	cosmosSenderSequence, err := strconv.ParseInt(string(claim.CosmosSenderSequence), 10, 64)
-	if err != nil {
-		sugaredLogger.Errorw("failed convert CosmosSenderSequence", errorMessageKey, err.Error())
-		return err
-	}
-	amount, err := strconv.ParseInt(claim.Amount, 10, 64)
-	if err != nil {
-		sugaredLogger.Errorw("failed convert Amount", errorMessageKey, err.Error())
-		return err
+	sugaredLogger.Infow(
+		"Sending new ProphecyClaim to CosmosBridge.",
+		"claim", claim,
+	)
+	// TODO this is leftover from when we were converting from a string; don't need it any more
+	var claimType ethbridge.ClaimType
+	switch claim.ClaimType {
```

Comment:
Like this todo says, we can probably change this away from a switch statement to a direct assignment like so:

`claimType = claim.ClaimType`

Reply:
Cleaned up this code and added a failure if the claim type isn't lock or burn